interactions/four_particle/cpp/integral.cpp

```diff
@@ -220,14 +211,14 @@ dbl F_f_vacuum_decay() {
     return -1;
 }
 
-dbl F_1_vacuum_decay() {
+dbl F_1_vacuum_decay(const std::array<dbl, 4> &f) {
     /* Constant part of the distribution functional */
-    return 0;
+    return f[3];
```

Comment:
This shouldn't be here

Reply:
True. But I don't know how else I should perform this test.

Reply:
Without vacuum decay approximation

Reply:
Unfortunately, this is not possible in the code, since the parts F_1 and F_f are different for reactions 
N --> 1 + 2 + 3 and 1 + 2 + 3 --> N.

Reply:
I don't get it. You are checking that the muons are being created? Just put the normal stuff in there and track the densities. Or is it a different test?

Reply:
F_1 for 1 + 2 + 3 --> N and F_f for N --> 1 + 2 + 3 are not the same. 
So I will be computing different quantities if I put back the normal stuff, which is not what we want.